### ISSUE TITLE
SULI Report: Abstract

### ISSUE THREAD

**Issue by sweverett** [drphilmarshall/Pangloss#79]
I need an abstract for the SULI report. The intended audience is another SULI intern; an undergraduate physics or science major with some level of research experience. This is what I came up with for my first draft:

"As dark matter does not absorb or emit light, its distribution in the universe must be inferred through indirect effects such as the gravitational lensing of distant galaxies. While most sources are only weakly lensed, the systematic alignment of background galaxies around a foreground lens can constrain the mass of the lens which is largely in the form of dark matter. In this paper, a simple halo model is used to approximate the dark matter mass distribution of the Millennium Simulation on the galaxy and cluster scales. Background sources are generated at a redshift of z = 1.3857 and then weakly lensed by the foreground halos contained in a light cone centered along each line of sight, which is carried out by the `Pangloss` framework. The model-predicted ellipticities of the background galaxies after lensing are compared to the "true" weakly-lensed ellipticities determined by the ray-traced convergence and shear calculated by Hilbert et al. (citation). Using the shear-shear and galaxy-galaxy correlation functions, I find that the `Pangloss` framework systematically under-predicts the shear power in both statistics and does not accurately capture the effect of dark matter structure at scales larger than 1 arcminute. Physical and computational shortcomings of the Pangloss framework are discussed, as well as potential improvements for upcoming work."

Would appreciate any feedback!

**Comment by drphilmarshall** [comment]
Good start. You can say that you have implemented a model for all the mass along the line of sight that is composed of a large number of individual halos associated with galaxies, and have (in this paper) checked it for accuracy using the ray-traced Millennium Simulation. You shoudl make clear that this a best-case test, since we assume knowledge of each halo mass. Finally: you shoudl quantify your results and include numbers in the abstract. By how much is the shear under-estimated and on what scales? What about the galaxy-galaxy lensing?

**Comment by sweverett** [comment]
Thanks for the feedback! I'll check some numbers later today and incorporate your comments.

By the way - DOE just changed the due date from Friday night to Thursday around 5:00. It'll be a busy couple days!

**Comment by drphilmarshall** [comment]
Good! That means you get Friday to tidy up code :-)

**Comment by sweverett** [comment]
That's the plan! I'll be adding comments, refining demos, and finishing up a few small issues both on Friday and in the upcoming weeks before we start the official post-SULI cleanup.

**Comment by sweverett** [comment]
Here's a second attempt at the abstract:

"As dark matter does not absorb or emit light, its distribution in the universe must be inferred through indirect effects such as the gravitational lensing of distant galaxies. While most sources are only weakly lensed, the systematic alignment of background galaxies around a foreground lens can constrain the mass of the lens which is largely in the form of dark matter. In this paper, I have implemented a framework to reconstruct all of the mass along lines of sight using a  best-case dark matter halo model in which the halo mass is known. This framework is then used to make predictions of the weak lensing of 3,240 generated source galaxies through a 0.09 deg^2 field of the Millennium Simulation. The lensed source ellipticities are characterized by the ellipticity-ellipticity and galaxy-mass correlation functions and compared to the same statistic for the intrinsic and ray-traced ellipticities. In the ellipticity-ellipticity correlation function, I find that the framework systematically under predicts the shear power on average by 117% and fails to capture correlation from dark matter structure at scales larger than 1 arcminute. The model predicted galaxy-mass correlation function is in agreement with the ray-traced statistic from scales 0.2 to 0.7 arcminutes, but systematically underpredicts shear power at scales larger than 0.7 arcminutes on average by 23.2%. Physical and computational shortcomings of the framework are discussed, as well as potential improvements for upcoming work."

**Comment by drphilmarshall** [comment]
Nice. That 117% is hard to parse, and over-precise. I'd go with "underpredict by a factor of 2.1" (or whatever it is) instead. 23.2% is also too many sig figs - it looks as though only 1 sf or maybe 2 is all that's justified given the size of the error bars. Also, can you express 0.09 sq deg as x by y sq arcmin instead please? Its a bit more information. You coudl also say "0.09 sq deg (X x X sq arcmin)" I think the current CPU time per background galaxy is also worth stating in the abstract, as well as a realistic projection for how much faster that could be made.

**Comment by sweverett** [comment]
Good suggestions! I'll incorporate those changes now.
